docs/azure/sdk/dependency-injection.md

```diff
@@ -108,6 +112,9 @@ In the [Register clients](#register-clients) section, you explicitly passed the
   "KeyVault": {
     "VaultUri": "https://mykeyvault.vault.azure.net"
   },
+  "ServiceBus": {
+    "Namespace": "<your-namespace>.servicebus.windows.net"
```

Comment:
Use underscore for consistency
```suggestion
    "Namespace": "<your_namespace>.servicebus.windows.net"
```

Reply:
Forgot about the config file ones, sorry about that.